community/kernel/src/test/java/org/neo4j/tooling/GlobalGraphOperationsIT.java

```diff
@@ -55,4 +67,139 @@ public void shouldListAllPropertyKeys() throws Exception
         }
     }
 
+    @Test
+    public void shouldReturnAllLabels()
+    {
+        // Given
+        GraphDatabaseService db = dbRule.getGraphDatabaseAPI();
+        Label dead = DynamicLabel.label( "dead" );
+        Label alive = DynamicLabel.label( "alive" );
+        try( Transaction tx = db.beginTx() )
+        {
+            db.createNode( alive );
+            db.createNode( dead );
+            tx.success();
+        }
+
+        try( Transaction tx = db.beginTx() )
+        {
+            Node node = IteratorUtil.single( db.findNodes( dead ) );
+            node.delete();
+            tx.success();
+        }
+
+        GlobalGraphOperations gg = GlobalGraphOperations.at( db );
+
+        // When - Then
+        try( Transaction _ = db.beginTx() )
```

Comment:
Right. Since Java 8, the use of underscore as an identifier is discouraged since they might want to give it special semantics in the future. And it is a compile time error to use them as parameter names in lambdas.